client/src/containers/explore/section/tile-menu/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from "react";
+
+import Link from "next/link";
+
+import { Card } from "@/components/ui/card";
+
+interface TileMenuProps {
+  items: { title: string; description: string; href: string }[];
+}
+
+const TileMenu: FC<TileMenuProps> = ({ items }) => {
+  return (
+    <div className="grid grid-cols-3 gap-0.5">
+      {items.map((i) => (
+        <Link key={`tile-menu-${i.href}`} href={i.href}>
+          <Card className="bg-primary p-6 hover:bg-secondary">
+            <h3 className="mb-6 font-semibold">{i.title}</h3>
```

Comment:
Instead of `mb-6` I would use `space-y-6` in the parent, but the outcome is the same.